x-pack/plugins/monitoring/public/components/cluster/overview/index.js

```diff
@@ -13,23 +13,32 @@ import { BeatsPanel } from './beats_panel';
 
 import { EuiPage, EuiPageBody } from '@elastic/eui';
 import { ApmPanel } from './apm_panel';
+import { UNLINKED_DEPLOYMENT_CLUSTER_UUID } from '../../../../common/constants';
 
 export function Overview(props) {
+  const isFromUnlinkedDeployment = props.cluster.cluster_uuid === UNLINKED_DEPLOYMENT_CLUSTER_UUID;
+
   return (
     <EuiPage>
       <EuiPageBody>
         <AlertsPanel alerts={props.cluster.alerts} changeUrl={props.changeUrl} />
 
-        <ElasticsearchPanel
-          {...props.cluster.elasticsearch}
-          version={props.cluster.version}
-          ml={props.cluster.ml}
-          changeUrl={props.changeUrl}
-          license={props.cluster.license}
-          showLicenseExpiration={props.showLicenseExpiration}
-        />
-
-        <KibanaPanel {...props.cluster.kibana} changeUrl={props.changeUrl} />
+        { !isFromUnlinkedDeployment ?
+          <ElasticsearchPanel
```

Comment:
If you put `()` then you could put the `KibanaPanel` in this same block and avoid duping the conditional.

Reply:
I'd need to wrap them in div or Fragment though, which I guess is fine

Reply:
Definitely prefer fragment over random `div`s appearing. :)